docs/stateful_models.md

```diff
@@ -0,0 +1,317 @@
+...
+
+SEQUENCE_START = 1
+SEQUENCE_END = 2
+sequence_id = 10
+
+
+inputs = {}
+
+""" 
+Add inputs with data to infer
+"""
+
+################   Add stateful specific inputs   #################
+
+################ Starting sequence with custom ID #################
+
+inputs['sequence_control_input'] = [int(SEQUENCE_START)]
+inputs['sequence_id'] = [int(sequence_id)]
+
+
+################   Starting sequence without ID   #################
+
+inputs['sequence_control_input'] = [int(SEQUENCE_START)]
+
+
+################       Non control requests       #################
+
+inputs['sequence_id'] = [int(sequence_id)]
+
+
+################         Ending sequence          #################
+
+inputs['sequence_control_input'] = [int(SEQUENCE_END)]
+inputs['sequence_id'] = [int(sequence_id)]
+
+###################################################################
+
+# Prepare request
+signature = "serving_default"
+request_body = json.dumps({"signature_name": signature,'inputs': inputs})
+
+# Send request to OVMS and get response
+response = requests.post("localhost:5555/v1/models/stateful_model:predict", data=request_body)
+
+# Parse response
+response_body = json.loads(response.text)
+
+# response_body variable now contains model outputs (inference results) as well as sequence_id in response_body["outputs"]
+
+# Fetch sequence id from the response
+sequence_id = response_body["outputs"]["sequence_id"]
+
+```
+See [rest_stateful_client.py](../example_client/rest_stateful_client.py) example client for reference.
+
+### Error codes
+
+When request is invalid or couldn't be processed you can expect following errors specific to inference on stateful models:
+
+| Description  | gRPC | HTTP |
+|---|---|---|
+| Sequence with provided ID does not exist. | NOT_FOUND | 404 NOT FOUND |
+| Sequence with provided ID already exists.  | ALREADY_EXISTS | 409 CONFLICT |
+| Server received SEQUENCE START request with ID of the sequence that is set for termination, but the last request of that sequence is still being processed. | FAILED_PRECONDITION | 412 PRECONDITION FAILED |
+| Max sequence number has been reached. Could not create new sequence. | UNAVAILABLE | 503 SERVICE UNAVAILABLE | 
+| Sequence ID has not been provided in request inputs. | INVALID_ARGUMENT | 400 BAD REQUEST |
+| Unexpected value of sequence control input. | INVALID_ARGUMENT | 400 BAD REQUEST |
+| Could not find sequence id in expected tensor proto field uint64_val. | INVALID_ARGUMENT | N/A |
+| Could not find sequence control input in expected tensor proto field uint32_val. | INVALID_ARGUMENT | N/A |
+| Special input proto does not contain tensor shape information. | INVALID_ARGUMENT | N/A |
+
+
+## Known limitations <a name="stateful_limitations"></a>
+
+There are following limitations when using stateful models with OVMS:
+
+ - Support for CPU only
+ - Support for Kaldi models with memory layers
+ - Support for non-Kaldi models with Tensor Iterator (with low latency transformation performed)
```

Comment:
Is there a link to OV docs?

Reply:
LowLatency transformation is mentioned in parameter description and there's a link to it there. 
Do you think I should add a link to it here? 
Or maybe merge those two points into one:
- Support Kaldi models with memory layers and non-Kaldi models with Tensor Iterator

And link the whole stateful networks doc in OV - https://docs.openvinotoolkit.org/latest/openvino_docs_IE_DG_network_state_intro.html ?
It seems more compact to me.